tools/rna_tools/intarna/test-data/intarna_result.csv

```diff
@@ -0,0 +1,5 @@
+id1;start1;end1;id2;start2;end2;subseqDP;hybridDP;E
```

Comment:
I'm not quit sure if this would work.
@martin-mann , will it cause a problem if I change the format?